desktop/mac/release-notes/index.md

```diff
@@ -29,9 +29,45 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed [CVE-2021-45449](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45449) that affects users currently on Docker Desktop version 4.3.0 or 4.3.1.
+
+### New
+
+- Easy, Secure sign in with Auth0 and Single Sign-on
+  - Single Sign-on: Users with a Docker Business subscription can now configure SSO to authenticate using their identity providers (IdPs) to access Docker. For more information, see [Single Sign-on](../../../single-sign-on/index.md).
+  - Signing in to Docker Desktop now takes you through the browser so that you get all the benefits of auto-filling from password managers.
+
+### Upgrades
+
+- [Docker Engine v20.10.12](https://docs.docker.com/engine/release-notes/#201012)
+- [Compose v2.2.3](https://github.com/docker/compose/releases/tag/v2.2.3)
+- [Kubernetes 1.22.5](https://github.com/kubernetes/kubernetes/releases/tag/v1.22.5)
+
+### Bug fixes and minor changes
+
+- It is now an error for `registry.json` to contain more than one organization in the `allowedOrgs` field. If you are using multiple organizations for different groups of developers, please provision a different `registry.json` file for each group.
+- Fixed the memory statistics for containers in the Dashboard. Fixes [docker/for-mac/#4774](https://github.com/docker/for-mac/issues/6076).
+- Added a deprecated option to `settings.json`: `"deprecatedCgroupsv1": true`, which switches the Linux environment back to cgroups v1. This option will be removed in future. If you have software which still requires cgroups v1, please update it to be compatible with cgroups v2.
+- Fixed a regression in Compose that reverted the container name separator from `-` to `_`. Fixes [docker/compose-switch](https://github.com/docker/compose-switch/issues/24).
+- Fixed an issue where sleeping the machine after pausing Docker Desktop, it could not be resumed after waking up. Fixes [for-mac#6058](https://github.com/docker/for-mac/issues/6058).
```

Comment:
```suggestion
- Fixed an issue where putting the machine to Sleep mode after pausing Docker Desktop results in a failure to resume the machine from the Sleep mode. Fixes [for-mac#6058](https://github.com/docker/for-mac/issues/6058).
```

Reply:
The machine wakes up all right, but Docker cannot resume from pause.
What about:
```suggestion
- Fixed an issue where putting the machine to Sleep mode after pausing Docker Desktop results in Docker Desktop not being able to resume from pause after the machines comes out of Sleep mode. Fixes [for-mac#6058](https://github.com/docker/for-mac/issues/6058).
```
?

Reply:
Sounds good, Mathieu.  Just one nit: after the **machine** comes out of Sleep...

Reply:
LGTM